src/driver_client/dclient.rs

```diff
@@ -115,6 +113,7 @@ impl DriverClient {
             CMS_ADDR::ADDR_CPU2HIF_CMS_INITIALIZE,
             1,
         )?;
+        sleep(Duration::from_millis(200));
```

Comment:
If there is no way around this, please add a comment explaining why.

Reply:
I ask Tony about sleep, because all shell function refer to his code

Reply:
If there is no way to avoid sleep, I suggest a detailed comment to avoid confusion later on.